README.md

```diff
@@ -0,0 +1,38 @@
+## Javascript Restaurant
+
+> A restaurant page created completely in Javascript using DOM manipulation and with the use of webpack.
+
+
+## Built With
+
+- Javscript
+- Webpack
+- HTML
+- CSS
+
+## Prerequisites
+
+- An up to date web browser such as Google Chrome, Mozilla Firefox, or Microsoft Edge
+
+## Setup
+- Clone or download repository to local machine
+- Open folder and click on index.html to have a local version of the app running
```

Comment:
Please add more information like to install packages via **npm install** and build it via **npm run build** and run it via **npm run start**